.github/workflows/docker.yml

```diff
@@ -2,8 +2,6 @@ name: Docker
 
 on:
   push:
-    tags:
-      - v**
```

Comment:
Why remove these? Do you just want to trigger this workflow manually?

Reply:
yes. just testing. reverted it back now.

Reply:
https://github.com/scroll-tech/scroll/actions/runs/9872663964
all passed here.